### DISCUSSION TITLE
Welcome to Aerie Discussions!

### DISCUSSION THREAD

**Discussion by patkenneally** [NASA-AMMOS/aerie#216] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tmikelow** [comment]
Hi All, It's Mike Low (previous Aerospace R2C2 now Auria Space).  I'm trying to find out if you have a hardened Java image from ironbank that you have used? Hope you al are well! Mike

**Reply by mattdailis** [reply]
Hi Mike! Welcome. To help organize our conversations, we like to make new discussion pages for new topics. Feel free to start a new conversation here: https://github.com/NASA-AMMOS/aerie/discussions/new?category=q-a

**Reply by ewferg** [reply]
Hey Mike! To answer your question directly the answer is no, we have not interacted with Iron Bank before. Happy to discuss further to see if there is anything we can do for you.

**Reply by ewferg** [reply]
@tmikelow, I had some time to play around this morning and was able to pull an Iron Bank java image down and run Aerie containers on top of that. If you want us to take you through what we did, just let us know (feel free to email for a more private conversation)

**Reply by tmikelow** [reply]
Hey Eric, Sorry it's taken so long.  I had a program Kickoff I needed to build and give.  Thanks for the information on getting Aerie running on an IronBank image.
